src/middlewared/middlewared/plugins/directoryservices_/cache.py

```diff
@@ -208,7 +208,7 @@ def refresh_impl(self, job):
         if ds['type'] is None:
             return
 
-        if ds['status'] != 'HEALTHY':
+        if ds['status'] not in (DSStatus.HEALTHY.name, DSStatus.JOINING.name):
```

Comment:
Line 103 above
```
                    name_key = 'group'
```
flake8 is complaining that the variable is assigned but never used.  When commented out it says the same thing about line 84.

Reply:
Just removed. Introduced in earlier commit.